launchpod/angular-launchpod/src/app/create-form/create-form.component.css

```diff
@@ -11,3 +11,7 @@ form {
 #message {
     color: red;
 }
+
+.mat-form-field {
+    width:30vw;
```

Comment:
If you want the form fields to be scaled to the size of the screen, you might want to consider doing that to the font as well. I could imagine use on a very large desktop screen -- the text inputs would still be 30% and 50% (for the result page) of the vw but the font might look too small in comparison. Maybe worth it to test on inspector by toggling mobile view, setting the device to responsive, and adjusting the screen resolution there.

Reply:
I inspected this and it seems like with different devices' screen sizes the font is still very readable!